remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/LifecycleManager.java

```diff
@@ -113,6 +115,7 @@ private void initProtobufMetadataManager(GlobalConfiguration globalCfg, GlobalCo
       SerializationContext serCtx = protobufMetadataManager.getSerializationContext();
       ClassLoader classLoader = globalCfg.classLoader();
       processProtostreamSerializationContextInitializers(classLoader, serCtx);
```

Comment:
I would expect the auto-loaded ProtostreamSerializationContextInitializers to be processed after the configured SerializationContextInitializers for more predictability in case of startup failures.

Reply:
Ok, updated